### PR TITLE
Fixed bugs and typos in Ukrainian translation

### PR DESCRIPTION
**Pull request by andriymoroz**

### CODE REVIEW

File: src/levels/mixed/jugglingCommits2.js [hunk 1]
```diff
@@ -1,512 +1,512 @@
-exports.level = {
```
Comment (pcottle) [inline src/levels/mixed/jugglingCommits2.js]: what happened here? do you  know why git shows the whole file as added/removed?

**Comment by pcottle** [comment]
Totally down to merge this except for the diff on `jugglingCommits2.js` and `tags.js`. any idea what went on there? did you change the newlines or something?

**Comment by pcottle** [comment]
yolo merge since i inspected locally and it looked fine

File: src/levels/mixed/jugglingCommits2.js (hunk 1 above)
Comment (andriymoroz) [inline src/levels/mixed/jugglingCommits2.js]: the line endings were in DOS format (CR+LF)
git even refused to apply its own patch created by format-patch if the patch touches these files

**Pull request merged**